workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/WorkflowGenerator.java

```diff
@@ -17,10 +17,14 @@
 
 import com.wipro.ats.bdre.BaseStructure;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wipro.ats.bdre.BaseStructure'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)